### PR TITLE
Conf delete

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-716

### Change description ###
cnp-flux-config: hmcts/cnp-flux-config#2866

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/bulk-scan-processor/values.aat.template.yaml [hunk 1]
```diff
@@ -5,3 +5,4 @@ java:
   devApplicationInsightsInstrumentKey: "f666440f-43c0-4abb-9c20-2ce175a374f0"
   environment:
     STORAGE_BLOB_SELECTED_CONTAINER: "ALL"
+    DELETE_COMPLETE_FILES_CRON: "0/10 * * * * *"
```
Comment (nitinprabhu) [inline charts/bulk-scan-processor/values.aat.template.yaml:8]: Is a reason we need this here. This config will only apply to staging

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -190,7 +190,8 @@ scheduling:
       cron: ${DELETE_REJECTED_FILES_CRON}
       ttl: ${DELETE_REJECTED_FILES_TTL}
     delete-complete-files:
-      enabled: false
+      enabled: ${DELETE_COMPLETE_FILES_ENABLED:false}
```
Comment (nitinprabhu) [inline src/main/resources/application.yaml:193]: we should remove default value

File: charts/bulk-scan-processor/values.aat.template.yaml (hunk 1 above)
Comment (apereverzin) [reply charts/bulk-scan-processor/values.aat.template.yaml]: Removed

File: charts/bulk-scan-processor/values.preview.template.yaml [hunk 3]
```diff
@@ -37,6 +37,7 @@ java:
     PROCESS_PAYMENTS_ENABLED: "false"
     OCR_VALIDATION_URL_BULKSCAN_SAMPLE_APP: "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
     DELETE_REJECTED_FILES_ENABLED: "false"
+    DELETE_COMPLETE_FILES_ENABLED: "false"
```
Comment (nitinprabhu) [inline charts/bulk-scan-processor/values.preview.template.yaml:40]: if we are running functional tests then we will need this enabled else functional test will fail

File: charts/bulk-scan-processor/values.aat.template.yaml (hunk 1 above)
Comment (nitinprabhu) [reply charts/bulk-scan-processor/values.aat.template.yaml:8]: we should not have any over rides here as this is only for staging

Comment (apereverzin) [reply charts/bulk-scan-processor/values.aat.template.yaml:8]: It is "0 0/10 * * * *" in values.yaml

**Pull request merged**